pallets/parachain-staking/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+authors = ["Manta Network"]
+description = "parachain staking pallet for collator selection and reward distribution"
+edition = "2021"
+homepage = 'https://manta.network'
+license = 'GPL-3.0'
+name = "pallet-parachain-staking"
+repository = 'https://github.com/Manta-Network/Manta/'
+version = "3.3.0"
+
+[dependencies]
+log = "0.4"
+serde = { version = "1.0.136", optional = true }
+
+# Substrate
+frame-benchmarking = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", optional = true, default-features = false }
+frame-support = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
+parity-scale-codec = { version = "3.1.2", default-features = false, features = ["derive"] }
+scale-info = { version = "2.1.2", default-features = false, features = ["derive"] }
+sp-arithmetic = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
+sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
+sp-std = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
+substrate-fixed = { git = "https://github.com/encointer/substrate-fixed.git", default-features = false }
+
+# Nimbus
+nimbus-primitives = { git = "https://github.com/manta-network/nimbus.git", branch = "manta-v3.3.0", default-features = false }
+
+# Manta
+manta-primitives = { path = '../../primitives/manta', default-features = false }
+# TODO: remove after whitelist period
```

Comment:
can we log this instead of the in-code todo

Reply:
I personally would rather have these as reminders in the code ( look at the size of our backlog :sweat_smile: )
Also, this allows easily finding relevant code by searching for "whitelist"

Reply:
I mean this is still part of our backlog just in a different place. Just put them for the next milestone and its probably more likely that you won't forget them

Reply:
this will be 4 months out, no idea what milestone that will be, the only place we currently have for that is the trashheap that's the `Future` milestone 😅 
If you think that's good enough I can remove the code comments, but I'm not convinced we'll resurface it there

Reply:
i have quite a few sections commented with `WHITELIST`, grepping the staking pallet for that helps surface parts of the code that have to do with this.

we can think about removing these in the followup #798